foreman-proxy-selinux-relabel

```diff
@@ -8,4 +15,9 @@
   /var/log/foreman-proxy \
   /var/run/foreman-proxy
 
+# Run hooks
+for HOOK in "$LIBEXEC_DIR/*-after-relabel.sh"; do
```

Comment:
The escaping needs to be done in a slightly different way, see [here](https://gist.github.com/adamruzicka/e15307a67309dd46250adc4c3fbe5cea)

Reply:
Hmmm good catch, I have tested this incorrectly.